configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
wait, let me make sure I understand. Are you saying that we should compute partial climatologies, say annual, cache those, them read them in and compute the full requested climo? That indeed would be similar to having to deal with yearly timeSeriesStatsClimatology data.